src/IMenuGraphr.ts

```diff
@@ -263,6 +263,11 @@ export interface IMenuSchema extends IMenuBase {
      * How the menu should be positioned within its container.
      */
     position?: IMenuSchemaPosition;
+
+    /**
+     * Whether or not the last selected index should be saved.
+     */
+    saveIndex?: boolean;
```

Comment:
What was the error? This property should really only belong to list menus, since there's no reason for it to belong to regular ones.